library/clientclass.py

```diff
@@ -81,3 +82,33 @@ def get_send_user_name(self):
     def get_type():
         """slack"""
         return 'slack'
+
+
+class ApiClient(BaseClient):
+    """
+    API形式のClient
+    Slackに対する操作は行わない
+    """
+
+    def __init__(self):
+        self.response = ''
+
+    def post(self, message):
+        """ポストする"""
+        self.response = os.linesep.join([self.response, message])
+
+    def upload(self, file, filename):
+        """ファイルを投稿する"""
+        self.response = os.linesep.join([self.response, f'upload: {filename}'])
+
+    def get_send_user(self):
+        """botを呼び出したユーザーを返す"""
+        return 'test_user_id'
```

Comment:
```suggestion
        return 'api_user_id'
```
`api_user_id` もおかしい気がするけど、 `test_user_id` は変な気がする。